src/monitor.cc

```diff
@@ -65,6 +69,43 @@ namespace ynode {
 // our singleton instance
 NodeMonitor* NodeMonitor::instance_ = NULL;
 
+// some utility functions to make code cleaner
+static inline v8::Local<v8::String> v8_str( const char* s ) {
+    Nan::MaybeLocal<v8::String> s_maybe = Nan::New<v8::String>(s);
+    return s_maybe.ToLocalChecked();
+}
+
+static inline v8::Local<v8::Value> getObjectProperty(const v8::Local<v8::Object>& object,
```

Comment:
Is "inline" enforced, or just a suggestion? (Will the compiler perhaps not inline if it decides that that's better?)